src/containers/DockerMocks.js

```diff
@@ -128,7 +128,7 @@ class BaseMock {
               }
               this.debug(`Mock - checking endpoint ${this.mockUrl} is online before proceed ...`)
               request(options, (err, response, body) => {
-                if (!err && response.statusCode === 200) {
+                if (!err && response.statusCode === container.caps[Capabilities.BOT_HEALTH_STATUS]) {
```

Comment:
I would suggest to do this check in the mocks itself - mocks only return 200 (OK) or 500 (NOT OK)